model_analyzer/result/result_manager.py

```diff
@@ -378,7 +368,7 @@ def _add_results_to_heaps(self, suppress_warnings=False):
                     model_name=model_name,
                     run_config=run_config,
                     comparator=self._run_comparators[model_name],
-                    constraints=self._run_constraints[model_name])
+                    constraints_holder=self._constraints_holder)
```

Comment:
old code passed in a single model constraints. We don't need to pass in all constraints here.